/jds/bin/diff -uprN gnome-panel-2.28.0.old/configure.in gnome-panel-2.28.0/configure.in
--- gnome-panel-2.28.0.old/configure.in	2009-09-21 17:35:48.000000000 +0100
+++ gnome-panel-2.28.0/configure.in	2009-10-15 15:52:34.875708828 +0100
@@ -174,7 +174,7 @@ PANEL_INPROCESS_ALL=
 PANEL_INPROCESS_APPLETS=
 AC_ARG_WITH(in-process-applets,
 	AC_HELP_STRING([--with-in-process-applets=APPLETS],
-		[comma-separated list of applets to compile in-process (possible values: none, clock, fish, notification-area, wncklet, all) @<:@default=none@:>@]),
+		[comma-separated list of applets to compile in-process (possible values: none, clock, notification-area, wncklet, all) @<:@default=none@:>@]),
 	[for i in `echo $withval | tr , ' '`; do
 		if test $i = "none"; then
 			PANEL_INPROCESS_NONE=1
@@ -182,20 +182,18 @@ AC_ARG_WITH(in-process-applets,
 			PANEL_INPROCESS_ALL=1
 		else if test $i = "clock"; then
 			PANEL_INPROCESS_APPLETS="$PANEL_INPROCESS_APPLETS $i"
-		else if test $i = "fish"; then
-			PANEL_INPROCESS_APPLETS="$PANEL_INPROCESS_APPLETS $i"
 		else if test $i = "notification-area"; then
 			PANEL_INPROCESS_APPLETS="$PANEL_INPROCESS_APPLETS $i"
 		else if test $i = "wncklet"; then
 			PANEL_INPROCESS_APPLETS="$PANEL_INPROCESS_APPLETS $i"
 		else
 			echo "applet $i not recognized, ignoring..."
-		fi; fi; fi; fi; fi; fi
+		fi; fi; fi; fi; fi
 	done],
 	[])
 
 if test -n "$PANEL_INPROCESS_ALL"; then
-	PANEL_INPROCESS_APPLETS="clock fish notification-area wncklet"
+	PANEL_INPROCESS_APPLETS="clock notification-area wncklet"
 fi
 if test -n "$PANEL_INPROCESS_NONE"; then
 	PANEL_INPROCESS_APPLETS=""
@@ -205,10 +203,6 @@ for i in $PANEL_INPROCESS_APPLETS; do
 		CLOCK_COMPILE_INPROCESS=1
 		AC_DEFINE([CLOCK_INPROCESS], 1,
 		[Defined when compiling the clock applet in-process])
-	else if test $i = "fish"; then
-		FISH_COMPILE_INPROCESS=1
-		AC_DEFINE([FISH_INPROCESS], 1,
-		[Defined when compiling the fish applet in-process])
 	else if test $i = "notification-area"; then
 		NOTIFICATION_AREA_COMPILE_INPROCESS=1
 		AC_DEFINE([NOTIFICATION_AREA_INPROCESS], 1,
@@ -332,7 +326,6 @@ po/Makefile.in
 applets/Makefile
 applets/clock/Makefile
 applets/clock/pixmaps/Makefile
-applets/fish/Makefile
 applets/notification_area/Makefile
 applets/wncklet/Makefile
 doc/Makefile
@@ -340,7 +333,6 @@ doc/reference/Makefile
 doc/reference/panel-applet/Makefile
 help/Makefile
 help/clock/Makefile
-help/fish/Makefile
 man/Makefile
 ])
 
/jds/bin/diff -uprN gnome-panel-2.28.0.old/applets/Makefile.am gnome-panel-2.28.0/applets/Makefile.am
--- gnome-panel-2.28.0.old/applets/Makefile.am	2009-05-02 02:01:58.000000000 +0100
+++ gnome-panel-2.28.0/applets/Makefile.am	2009-10-15 15:52:34.875323844 +0100
@@ -1,6 +1,5 @@
 SUBDIRS =		\
 	clock		\
-	fish		\
 	notification_area \
 	wncklet
 
/jds/bin/diff -uprN gnome-panel-2.28.0.old/help/Makefile.am gnome-panel-2.28.0/help/Makefile.am
--- gnome-panel-2.28.0.old/help/Makefile.am	2009-07-01 14:06:23.000000000 +0100
+++ gnome-panel-2.28.0/help/Makefile.am	2009-10-15 15:52:34.875929591 +0100
@@ -1,3 +1,3 @@
-SUBDIRS = clock fish
+SUBDIRS = clock
 
 -include $(top_srcdir)/git.mk
diff -ruN gnome-panel-2.29.92.1.orig/configure.in gnome-panel-2.29.92.1/configure.in
--- gnome-panel-2.29.92.1.orig/configure.in	2010-03-15 03:26:34.022622316 +0000
+++ gnome-panel-2.29.92.1/configure.in	2010-03-15 03:27:00.996859771 +0000
@@ -213,7 +213,7 @@
 		WNCKLET_COMPILE_INPROCESS=1
 		AC_DEFINE([WNCKLET_INPROCESS], 1,
 		[Defined when compiling the wncklet applet in-process])
-	fi; fi; fi; fi
+	fi; fi; fi
 done
 if test "x$PANEL_INPROCESS_APPLETS" = "x"; then
 	PANEL_INPROCESS_APPLETS="(none)"
